index/slashing/slashing.go

```diff
@@ -243,9 +243,8 @@ func epochPatch(ctx context.Context, c API, pts *types.TipSet, ts *types.TipSet)
 			}
 			mas, ok := as.State.(map[string]interface{})
 			if !ok {
-				panic("read state should be a map interface result")
+				log.Debugf("read state should be a map interface result: %#v", as.State)
```

Comment:
For some reason this case started happening. Opted to not be fatal, log, and move on.
Basically is returning `nil`, but not a typed `map[string]interface{}` nil or empty thing.